client/extensions/woocommerce/state/ui/payments/methods/README.md

```diff
@@ -0,0 +1,58 @@
+Payment Methods
+================
+
+This module is used to manage payment method related UI.
+
+## Actions
+
+### `openPaymentMethodForEdit( siteId: number, id: string )`
```

Comment:
This documentation looks a lot like JSDoc. I wonder what the long-term effects will be of maintaining documentation both here and inline with JSDoc in this way.